Build failing on Windows

Luarocks xml 1.1.0-1 fails to build on Windows due to a libstdc++ dependency. It should be OK to exclude this dependency on Windows since the runtime is linked by default.

Here's the log:

    D:\>luarocks install xml
    Installing http://luarocks.org/repositories/rocks/xml-1.1.0-1.src.rock...
    Using http://luarocks.org/repositories/rocks/xml-1.1.0-1.src.rock... switching to 'build' mode
    cl /MD /O2 -c -Fosrc/bind/dub/dub.obj -ID:/luarocks/2.1/include src/bind/dub/dub.cpp -Iinclude -Isrc/bind -Isrc/vendor
    Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 16.00.40219.01 for 80x86
    Copyright (C) Microsoft Corporation.  All rights reserved.

    dub.cpp
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used,
     but unwind semantics are not enabled. Specify /EHsc
    cl /MD /O2 -c -Fosrc/bind/xml_core.obj -ID:/luarocks/2.1/include src/bind/xml_core.cpp -Iinclude -Isrc/bind -Isrc/vendor

    Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 16.00.40219.01 for 80x86
    Copyright (C) Microsoft Corporation.  All rights reserved.

    xml_core.cpp
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used,
     but unwind semantics are not enabled. Specify /EHsc
    cl /MD /O2 -c -Fosrc/bind/xml_Parser.obj -ID:/luarocks/2.1/include src/bind/xml_Parser.cpp -Iinclude -Isrc/bind -Isrc/ve
    ndor
    Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 16.00.40219.01 for 80x86
    Copyright (C) Microsoft Corporation.  All rights reserved.

    xml_Parser.cpp
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used,
     but unwind semantics are not enabled. Specify /EHsc
    cl /MD /O2 -c -Fosrc/Parser.obj -ID:/luarocks/2.1/include src/Parser.cpp -Iinclude -Isrc/bind -Isrc/vendor
    Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 16.00.40219.01 for 80x86
    Copyright (C) Microsoft Corporation.  All rights reserved.

    Parser.cpp
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\INCLUDE\xlocale(323) : warning C4530: C++ exception handler used,
     but unwind semantics are not enabled. Specify /EHsc
    link -dll -def:core.def -out:xml/core.dll D:/luarocks/2.1/lua5.1.lib src/bind/dub/dub.obj src/bind/xml_core.obj src/bind
    /xml_Parser.obj src/Parser.obj stdc++.lib
    Microsoft (R) Incremental Linker Version 10.00.40219.01
    Copyright (C) Microsoft Corporation.  All rights reserved.

    LINK : fatal error LNK1181: cannot open input file 'stdc++.lib'

    Error: Build error: Failed compiling module xml/core.dll


Cheers,
Darren

--- a/xml-1.1.1-1.rockspec
+++ b/xml-1.1.1-1.rockspec
@@ -40,8 +40,16 @@ build = {
         'src/bind/xml_core.cpp',
       },
       incdirs   = {'include', 'src/bind', 'src/vendor'},
-      libraries = {'stdc++'},
     },
   },
+  -- Override default build options
+  platforms = {
+    unix = { -- for all unix system like linux, macos, bsd, cygwin...
+      modules = {
+        ['xml.core'] = {
+          libraries = {'stdc++'}
+        }
+      }
+    }
+  }
 }
-

